lib/search/widgets/search_widget.dart

```diff
@@ -0,0 +1,158 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:mi_libro_vecino/search/cubit/search_cubit.dart';
+import 'package:mi_libro_vecino/ui_utils/colors.dart';
+
+class SearchWidget extends StatefulWidget {
+  SearchWidget({Key? key}) : super(key: key);
+
+  final TextEditingController textEditControler = TextEditingController();
+
+  @override
+  SearchWidgetState createState() => SearchWidgetState();
+}
+
+class SearchWidgetState extends State<SearchWidget> {
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<SearchCubit, SearchState>(
+      builder: (context, state) {
+        return AnimatedContainer(
+          width: 800,
+          height: state is SearchQueryChanged
+              ? 85 + 50.5 * state.suggestions.length - 1
```

Comment:
here we can change the 'formula' with a named variable in order to increase readability